smoke/features/csi-driver-e2e.feature

```diff
@@ -12,13 +12,16 @@ Feature: SharedSecrets and SharedConfigMap
     @automated
     Scenario: Configmaps are accessbile throughout the cluster : CSI-01-TC01
```

Comment:
I would mention in the scenario description here that you have included the refreshResources setting validation

Reply:
You missed making this update in your last commit @prietyc123

Reply:
Done